services/provider/server/server.go

```diff
@@ -349,7 +356,14 @@ func (s *OCSProviderServer) GetDesiredClientState(ctx context.Context, req *pb.G
 	case ocsv1alpha1.StorageConsumerStateDeleting:
 		return nil, status.Errorf(codes.NotFound, "StorageConsumer is in deleting phase")
 	case ocsv1alpha1.StorageConsumerStateReady:
-		kubeResources, err := s.getKubeResources(ctx, consumer)
+
+		consumerConfig, err := s.getConsumerConfig(ctx, consumer.Status.ResourceNameMappingConfigMap.Name, consumer.Namespace)
```

Comment:
Is it a user editable cm?
Or is it controlled by storageconsumer?

Reply:
its controlled by storage consumer